genotype_api/services/endpoint_services/sample_service.py

```diff
@@ -78,30 +60,35 @@ def _get_sample_response(self, sample: Sample) -> SampleResponse:
         )
 
     def get_sample(self, sample_id: str) -> SampleResponse:
-        sample: Sample = get_sample(session=self.session, sample_id=sample_id)
+        sample: Sample = self.store.get_sample(sample_id=sample_id)
         if not sample:
             raise SampleNotFoundError
         if len(sample.analyses) == 2 and not sample.status:
-            sample: Sample = refresh_sample_status(session=self.session, sample=sample)
+            sample: Sample = self.store.refresh_sample_status(sample=sample)
         return self._get_sample_response(sample)
 
     def get_samples(self, filter_params: SampleFilterParams) -> list[SampleResponse]:
-        samples: list[Sample] = get_filtered_samples(
-            session=self.session, filter_params=filter_params
-        )
+        samples: list[Sample] = self.store.get_filtered_samples(filter_params=filter_params)
         return [self._get_sample_response(sample) for sample in samples]
 
-    def create_sample(self, sample: Sample) -> None:
-        create_sample(session=self.session, sample=sample)
+    def create_sample(self, sample: SampleCreate) -> None:
```

Comment:
The naming of this parameter is confusing, would you consider renaming it?
```suggestion
    def create_sample(self, sample_create: SampleCreate) -> None:
```